tests/test_autodiff.py

```diff
@@ -15,5 +90,46 @@ def test_functions(test_obj):
     # Check that outputs match within margin
     np.testing.assert_allclose(direct_output, interpreted_output, rtol=1e-10, atol=1e-10)
 
+@hypothesis.given(get_test_samples([
+    TestFunc(tensor_courpus_add, elementwise_strategy(2, values=[
+        st.just(-1.0),  # Fixed value for first argument
+        st.just(1.0)    # Fixed value for second argument
+    ]))
+]))
+def test_jacobians(test_obj):
+    """Test that jacobian computation matches finite difference approximation"""
+    func, args, static_argnames = test_obj
+    print(f"\nTesting function: {func.__name__}")
+    print(f"Args shapes: {[arg.shape if hasattr(arg, 'shape') else arg for arg in args]}")
+    print(f"Args values: {[arg if not hasattr(arg, 'shape') else 'array' for arg in args]}")
+
+    # Create grad transform with Gradient configured for full jacobian
+    grad_transform = grad(func, static_argnames=static_argnames)
+    # Override the default gradient_only=True in the first transform (Gradient)
+    grad_transform.transforms = (Gradient(gradient_only=False),) + grad_transform.transforms[1:]
```

Comment:
You should just plumb this through `grad` instead of doing this if the functionality is missing